### PR TITLE
PHP added

### PR DESCRIPTION
**Pull request by Aswin-2002**
Implemented PHP in sign up form to enter user data into database(mysql)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h1>Project overview</h1>
       </div>
       <div class="content hidden">
         <h1>Development Process</h1>
-        <p>We utilized Python and Django for the backend,<br>
```
Comment (Aswin74) [inline about.html:81]: test comment

File: about.html [hunk 2]
```diff
@@ -78,7 +78,7 @@ <h1>Project overview</h1>
       </div>
       <div class="content hidden">
         <h1>Development Process</h1>
-        <p>We utilized Python and Django for the backend,<br>
+        <p>We utilized PHP for the backend,<br>
```
Comment (Aswin74) [inline about.html:81]: test2

File: about.html (hunk 1 above)
Comment (Aswin-2002) [reply about.html:81]: hii

Comment (Aswin74) [reply about.html:81]: test2

File: login.html [hunk 3]
```diff
@@ -1,108 +1,110 @@
...
+        preferred college.<br> </p>
+      <p>Thank you for considering us for your hostel booking.</p>
+    </div>
+
+    <!--login/signup-->
+    <div class="form hidden col-lg-6 me-lg-4 mt-sm-4">
+
+      <input type="checkbox" id="hidden-btn" /><!--trigger-->
+
+      <form action="signup.php" method="post">
```
Comment (Aswin74) [inline login.html:83]: form

Comment (Aswin-2002) [inline login.html:83]: this line

**Review by Aswin74 — APPROVED** [review]
php

**Pull request merged**